empyrical/stats.py

```diff
@@ -140,11 +142,14 @@ def cum_returns(returns, starting_value=0):
     if len(returns) < 1:
         return type(returns)([])
 
-    if np.isnan(np.asanyarray(returns)[0]):
-        returns = returns.copy()
-        returns[0] = 0.
+    if isinstance(returns, pd.DataFrame):
+        df_cum = (returns.fillna(0) + 1).cumprod()
 
-    df_cum = np.exp(nancumsum(np.log1p(returns)))
+    else:
+        if np.all(np.isnan(np.asanyarray(returns)[0])):
```

Comment:
If one of the returns streams has a nan at the start, do we not want to replace it with a zero, since this is the case for a 1D array?